assignments/callbacks.js

```diff
@@ -81,20 +81,25 @@ console.log(multiplyNums(4, 3, pint));
 // multiplyNums multiplies two numbers and passes the result to the callback.
 
 
-function contains(item, list, cb) {
-  return cb(item, list);
+function contains(item, list, cb){
+  let result;
+  list.includes(item) ? result = true : result = false; //turnary is if/else
+
+  cb(result);
 }
 
-const find = function(item, list){
-  if (list.contains(item)){
-    return true;
+const find = function(result){
+ console.log(result);
 }
```

Comment:
should return result instead of console.log here and then console log the contains funtion call.  remember functions take an input, does something with that input and returns an output